pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,221 @@
+
+<CodeBlock>
+{`asyncapi: '3.0.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+servers:
+  mosquitto:
+    host: test.mosquitto.org
+    protocol: mqtt
+channels:
+  lightMeasured:
+  	address: 'light/measured'
+	messages:
+		lightMeasuredMessage:
+			name: LightMeasured
+			payload:
+				type: object
+				properties:
+					id:
+						type: integer
+						minimum: 0
+						description: Id of the streetlight.
+					lumens:
+						type: integer
+						minimum: 0
+						description: Light intensity measured in lumens.
+					sentAt:
+						type: string
+						format: date-time
+						description: Date and time when the message was sent.
+operations:
+	onLightMeasured:
+		action: 'recieve'
+		summary: Inform about enviromental lighting conditions for a particular streetlight.
+		operationId: onLightMeasured
+		channel:
+			$ref: '#/channels/lightMeasured'`}
+</CodeBlock>
+
+Let's break it down into pieces:
+
+<CodeBlock>
+{`asyncapi: '3.0.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+`}
+</CodeBlock>
+
+- The `asyncapi` field indicates you use the AsyncAPI version 3.0.0.
+
+- The `info` section holds information about the Streetlights API. Here, its
+title, version, description and license is defined.
+
+Moving on, let's talk about the `servers` section. In this section, you define
+the connection details of the servers that will handle the events sent by
+the streetlights to your API. Each server's connection detail is
+defined as a Server Object:
+<CodeBlock>
+{`servers:
+  	mosquitto:
+  		host: test.mosquitto.org
+    	protocol: mqtt
+`}
+</CodeBlock>
+
+You have defined a Server Object named **mosquitto**, which
+refers to the Eclipse Mosquitto message broker. In a Server Object, the `host`
+field is used to specify the hostname of the server, and the `protocol` field is
+used to specify the protocol the server supports. The `host` is
+set as a real instance of the broker
+[hosted by the Mosquitto community](https://test.mosquitto.org/) and the
+`protocol` is set as **MQTT**. If you do not want to use the test instance, you can run
+a broker on your local machine with `docker run -it -p 1883:1883 eclipse-mosquitto:1.5`.
+But remember to change `host` to `localhost:1883`
+
+Now lets move on to the `channels` section. In this section, you define the
+communication channels for all the events that will be processed by the message
+broker. Each communication channel is defined as a Channel Object with an
+identifier that follow common programming naming conventions:
```

Comment:
![Screenshot from 2023-07-22 07-46-52](https://github.com/asyncapi/website/assets/99868037/2614129c-dbc1-4476-9bb9-5a291217a02c)

The original code block was divided into two section but you merged into one. Reason?